core/spec/dummy/app/models/author.rb

```diff
@@ -1,7 +1,3 @@
 class Author < ActiveRecord::Base
   has_many :blogs
-
-  def to_s
-    name
-  end
```

Comment:
:+1:

Might be good to document this behavior somewhere? Or at least make it understood that SimpleForm is responsible for this.